code/modules/events/scrubber_clog.dm

```diff
@@ -0,0 +1,193 @@
+/datum/round_event_control/scrubber_clog
+	name = "Minor Scrubber Clog"
+	typepath = /datum/round_event/scrubber_clog
+	weight = 25
+	max_occurrences = 3
+	earliest_start = 5 MINUTES
+
+/datum/round_event/scrubber_clog
+	announceWhen = 10
+	startWhen = 5
+	endWhen = 600
+
+	///Scrubber selected for the event.
+	var/obj/machinery/atmospherics/components/unary/vent_scrubber/scrubber
+	///What mob will be spawned
+	var/mob/spawned_mob = /mob/living/basic/cockroach
+	///Cap on the number of spawned mobs that can be alive at once.
+	var/maximum_spawns = 3
+	///Interval between mob spawns.
+	var/spawn_delay = 10
+	///Used to track/limit produced mobs.
+	var/list/living_mobs = list()
+	///Used for tracking if the clog signal should be sent.
+	var/clogged = TRUE
+
+/datum/round_event/scrubber_clog/announce()
+	priority_announce("Minor biological obstruction detected in the ventilation network. Blockage is believed to be in the [get_area(scrubber)] area.", "Custodial Notification")
+
+/datum/round_event/scrubber_clog/setup()
+	scrubber = get_scrubber()
+	if(!scrubber)
+		CRASH("Unable to find suitable scrubber.")
+
+	RegisterSignal(scrubber, COMSIG_PARENT_QDELETING, .proc/scrubber_move)
+
+	spawned_mob = get_mob()
+	endWhen = rand(300, 600)
+	maximum_spawns = rand(3, 5)
+	spawn_delay = rand(10, 15)
+
+/datum/round_event/scrubber_clog/start() //Sets the scrubber up for unclogging/mob production.
+	scrubber.clog()
+	scrubber.produce_mob() //The first one's free!
```

Comment:
You aren't passing any arguments into `produce_mob()` here, and it expects some from what I can tell?